druid-shell/src/platform/windows/error.rs

```diff
@@ -31,10 +40,42 @@ pub enum Error {
     NullHwnd,
 }
 
+#[allow(clippy::crosspointer_transmute)]
+fn hresult_description(hr: HRESULT) -> Option<String> {
+    unsafe {
+        let mut message_buffer: LPWSTR = std::ptr::null_mut();
+        let format_result = FormatMessageW(
+            FORMAT_MESSAGE_FROM_SYSTEM
+                | FORMAT_MESSAGE_ALLOCATE_BUFFER
+                | FORMAT_MESSAGE_IGNORE_INSERTS
+                | FORMAT_MESSAGE_MAX_WIDTH_MASK,
+            null(),
+            hr as u32,
```

Comment:
Let's change this `u32` to `DWORD` (from `winapi::shared::minwindef`). I know functionally they're the same and the `u32` looks nicer but matching the types used in Microsoft docs when calling a winapi function will help whomever is reading this code in the future.

Reply:
Comepletely agree, changed it